src/Tizen.Content.Download/Tizen.Content.Download/CacheManager.cs

```diff
@@ -0,0 +1,162 @@
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using System.ComponentModel;
+
+namespace Tizen.Content.Download
+{
+    /// <summary>
+    /// The CacheManager class provides the functions to manage cache properties.
+    /// </summary>
+    /// <since_tizen> 7.5 </since_tizen>
+    static class CacheManager
+    {
+
+        /// <summary>
+        /// Clears the cache.
+        /// </summary>
+        /// <since_tizen> 7.5 </since_tizen>
+        /// <privilege>http://tizen.org/privilege/download</privilege>
+        /// <exception cref="ArgumentException">Thrown when it is failed due to an invalid parameter.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when it is failed due to an invalid operation.</exception>
+        /// <exception cref="UnauthorizedAccessException">Thrown when a permission is denied.</exception>
+        /// <exception cref="NotSupportedException">Thrown when not supported.</exception>
+        static public void ResetCache()
+        {
+            int ret = Interop.Download.ResetDownloadCache();
+            if (ret != (int)DownloadError.None)
+            {
+                DownloadErrorFactory.ThrowException(ret, "Failed to reset cache");
+            }
+        }
+
+        /// <summary>
+        /// Max size(MB) of the cache.
+        /// </summary>
+        /// <since_tizen> 7.5 </since_tizen>
+        /// <privilege>http://tizen.org/privilege/download</privilege>
+        /// <exception cref="ArgumentException">Thrown when it is failed due to an invalid parameter.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when it is failed due to an invalid operation.</exception>
+        /// <exception cref="UnauthorizedAccessException">Thrown when a permission is denied.</exception>
+        /// <exception cref="NotSupportedException">Thrown when not supported.</exception>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        static public uint MaxCacheSize
+        {
+            get
+            {
+                uint maxCacheSize;
+                int ret = Interop.Download.GetDownloadCacheMaxSize(out maxCacheSize);
+                if (ret != (int)DownloadError.None)
+                {
+                    DownloadErrorFactory.ThrowException(ret, "Failed to get max size of cache");
+                }
+                return maxCacheSize;
+            }
+            set
+            {
+                int ret = Interop.Download.SetDownloadCacheMaxSize(value);
+                if (ret != (int)DownloadError.None)
+                {
+                    DownloadErrorFactory.ThrowException(ret, "Failed to set cache max size");
+                }
+            }
+        }
+
+        /// <summary>
+        /// Clears all cache.
```

Comment:
Please say more about ResetCache() and ResetAllCache() - do they apply to user cache, all system download cache, per app download cache?

Reply:
Updated.